client/src/features/update/update-model.ts

```diff
@@ -150,14 +152,16 @@ export class UpdateModelCommand extends Command {
                 // An element has been removed
                 const left = match.left
                 if (isFadeable(left) && match.leftParentId !== undefined) {
-                    const parent = newRoot.index.getById(match.leftParentId)
-                    if (parent instanceof SParentElement) {
-                        const leftCopy = context.modelFactory.createElement(left) as SChildElement & Fadeable
-                        parent.add(leftCopy)
-                        animationData.fades.push({
-                            element: leftCopy,
-                            type: 'out'
-                        })
+                    if (newRoot.index.getById(left.id) === undefined) {
```

Comment:
When a parent and a child are removed, the child would be added twice by the update (with the parent and with its own match), thus causing duplicate IDs in the newRoot.